src/less/listbox/base/listbox.less

```diff
@@ -48,8 +51,17 @@ div.listbox__option svg.icon {
 }
 
 div.listbox__options[role="listbox"]:focus .listbox__option--active[role="option"] {
-    .background-color(listbox-option-hover-background-color);
-    .color(listbox-option-hover-foreground-color);
+    background-color: var(
+        --listbox-option-hover-background-color,
+        var(
+            --color-selection-list-item-hover-background,
+            @color-selection-list-item-hover-background
+        )
+    );
```

Comment:
I think this would be my only concern, but I think a mixin would help.
Like:

`.background-color(listbox-option-hover-background-color, color-selection-list-item-hover-background);`

And in the mixin, we generate it if it has a second argument or not.
This way any code we don't change will just need a second argument and we can leave the other mixins as is.

Reply:
Looking at the code, this should be pretty easy.

```
.background-color(@token) {
    .customProperty(background-color, @token);
}
.background-color(@token, @defaultToken) {
    .customProperty(background-color,   @token, .customProperty(background-color, @defaultToken));
}
```

Something like that should work. Again, didn't test it but this seems to be working well, and that requires the least amount of code change in less files.

Reply:
I also don't like the way prettier formats it across multiple lines. I find it a lot easier to read on one line. But yes, I will definitely look to simplify this with a mixin before going any further. Cheers.

Reply:
Here's what I have:

![Screen Shot 2021-09-09 at 10 30 06 AM](https://user-images.githubusercontent.com/38065/132734299-c998cfc3-c790-4a1a-8676-be8e5e289626.png)

![Screen Shot 2021-09-09 at 10 30 15 AM](https://user-images.githubusercontent.com/38065/132734318-d5b1029f-6e82-4c26-8301-db9743df30b1.png)

I've had to append the work "token" to these mixin names for now just so they don't clash with the existing ones.

Reply:
Yes that's probably fine for now.